pkgs/tools/networking/pritunl-client/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "pritunl-client";
+  version = "1.3.3300.95";
+
+  src = fetchFromGitHub {
+    owner = "pritunl";
+    repo = "pritunl-client-electron";
+    rev = version;
+    sha256 = "sha256-kPZFfKdiIq45/uyfsllUJrCphfUV5VX5QwaEg8351GI=";
+  };
+
+  modRoot = "cli";
+  vendorSha256 = "sha256-fI2RIzvfbqBgchsvY8hsiecXYItM2XX9h8oiP3zmfTA=";
+
+  postInstall = ''
+    mv $out/bin/cli $out/bin/pritunl-client
+  '';
+
+  meta = with lib; {
+    description = "Pritunl OpenVPN client CLI";
+    homepage = "https://github.com/pritunl/pritunl-client-electron/tree/master/cli";
```

Comment:
```suggestion
    homepage = "https://github.com/pritunl/pritunl-client-electron/";
```

Reply:
I thought it was the same as [[gopls](https://github.com/golang/tools/tree/master/gopls)](https://github.com/golang/tools/tree/master/gopls) where the repo is `tools` but the package is in subdirectory `gopls`.

https://github.com/NixOS/nixpkgs/blob/c5924154f000e6306030300592f4282949b2db6c/pkgs/development/tools/gopls/default.nix#L24

Reply:
Well, I never have seen such a thing before.
Monorepos containing many different programs I have seen, but not this specific code on Nixpkgs.
Ok then.

Reply:
So can I resolve this? or should I still need to change it?